netstandard/ref/System.Net.NetworkInformation.cs

```diff
@@ -315,10 +315,16 @@ public partial class NetworkAvailabilityEventArgs : System.EventArgs
         internal NetworkAvailabilityEventArgs() { }
         public bool IsAvailable { get { throw null; } }
     }
-    public static partial class NetworkChange
+    public partial class NetworkChange
```

Comment:
Why do we want to make this backwards-looking change in new version?

Reply:
Agreed we should keep this static.

Reply:
And remove the obsolete constructor.

Reply:
Fixed